lib/logger.js

```diff
@@ -70,7 +70,7 @@ const serializers = {
 }
 
 function now () {
-  var ts = process.hrtime()
+  const ts = process.hrtime()
```

Comment:
This function is run in a hot path (i.e. every request). It was likely a `var` for this reason. However, at this point, I don't think the performance difference is so great.

https://jsperf.com/let-vs-var-performance/67